docs/transaction-api.asciidoc

```diff
@@ -73,6 +73,55 @@ When a span is started it will measure the time until <<span-end,`span.end()`>>
 
 See <<span-api,Span API>> docs for details on how to use custom spans.
 
+[[transaction-set-tag]]
+==== `transaction.setTag()`
+
+[source,js]
+----
+transaction.setTag(name, value)
+----
+
+Set a tag on the transaction.
+You can set multiple tags on the same transaction.
+If an error happens during the transaction,
+it will also get tagged with the same tags.
+
+Tags are key/value pairs that are indexed by Elasticsearch and therefore searchable (as opposed to data set via <<apm-set-custom-context,`apm.setCustomContext()`>>).
+
+Arguments:
+
+* `name` - Any string.
+Periods (`.`), asterisks (`*`), and double quotation marks (`"`) will be replaced by underscores (`_`),
+as those characters have special meaning in Elasticsearch
+* `value` - Any string.
+If a non-string data type is given,
+it's converted to a string before being sent to the APM Server
+
+[[transaction-add-tags]]
+==== `transaction.addTags()`
+
+[source,js]
+----
+apm.addTags({ [name]: value })
```

Comment:
Is the extra whitespace in this line by design? Looks kinda goofy to me. Your call:
<img width="341" alt="screen shot 2018-12-18 at 12 29 00 pm" src="https://user-images.githubusercontent.com/5618806/50181184-859ac580-02c0-11e9-9a15-b00913f7e4f0.png">

Reply:
Technically yes. But it's not super intuitive. In general I'd like to discuss in general how to better show the method signature as it's currently done in several different ways throughout the documentation with no common logic. I'll create a separate issue for it and mark this review comment as resolved.

Reply:
Here: #760

Reply:
Perfect! Thanks